cmd/kots/cli/admin-console-generate-manifests.go

```diff
@@ -73,6 +73,7 @@ func AdminGenerateManifestsCmd() *cobra.Command {
 				IsOpenShift:          isOpenShift,
 				IsGKEAutopilot:       isGKEAutopilot,
 				RegistryConfig:       registryConfig,
+				TrustedCAsConfigmap:  v.GetString("private-ca-configmap"),
```

Comment:
why the naming inconsistency? how about:
```suggestion
				PrivateCAsConfigmap:  v.GetString("private-ca-configmap"),
```